### PR TITLE
add cdf_posteriori to return M, MCMC samples

### PR DESCRIPTION
**Pull request by mbratten**
Added `cdf_posteriori` function so that the `M` cdf trace is available in full rather then only as its mean, and high and low CI estimates.

### CODE REVIEW

**Comment by stphnma** [comment]
this is pretty cool! definitely useful to get the full posterior samples

File: convoys/regression.py [hunk 1]
```diff
@@ -278,20 +278,16 @@ def callback(LL, value_history=[]):
             'beta': data[6+n_features:6+2*n_features].T,
         } for k, data in result.items()}
 
-    def cdf(self, x, t, ci=None):
+    def cdf_posteriori(self, x, t, ci=None):
```
Comment (erikbern) [inline convoys/regression.py]: Is this a name that's used elsewhere? Was thinking it's not really a posteriori distribution, but samples from the posterior distribution – maybe pick a name to make that clear?

**Comment by erikbern** [comment]
Nice!

File: convoys/regression.py (hunk 1 above)
Comment (mbratten) [reply convoys/regression.py]: Yeah I think thats fair, it is the samples themselves.

Comment (erikbern) [reply convoys/regression.py]: We can rename it at some other point. I think the "cdf" naming is really confusing already – should probably just be called `predict` or something to be consistent with scikit-learn.

**Pull request merged**